### PR TITLE
Add support for fallback locale

### LINKED ISSUE #10: [FEATURE] Support for fallback locales
Ad the title says, support the Laravel fallback locale, right now is only searching in the main locale

### PR DESCRIPTION
**Pull request by sergix44**
- Adds support for fallback locale
- Added how to configure Lingua when using Svelte
- Added reasonable defaults to the config object

Closes #10

### CODE REVIEW

**Review by mikield — CHANGES_REQUESTED** [review]
I think we can provide such Svelte component, so users will just import it from our package.
We also can provide up to date changes for that component.

**Comment by sergix44** [comment]
> I think we can provide such Svelte component, so users will just import it from our package. We also can provide up to date changes for that component.

Yes, but also no:

- Since Svelte is not supported by Laravel directly (there is no direct preset like for vue or react for inertia), the setup is highly dependant on how the dev has decided prefixes, aliases, etc.
- That would mean splitting up the index.js by frameworks, in my tests I had to import the trans function from the translator.js instead of the index.js, because I was getting an error because vue was not there (of course)

So basically depends on how much opinionated you want to make Lingua

**Comment by mikield** [comment]
Yeap, that's a problem for future extension for this package.
I think I need to remove `index.js` totally, and anyway people will need to add an alias into their `vite.config.js`, we can provide something like:

### Vue:
```js
'lingua-js': resolve(__dirname, 'vendor/cyberwolfstudio/lingua/dist/vue.js')
```

### Svelte 
```js
'lingua-js': resolve(__dirname, 'vendor/cyberwolfstudio/lingua/dist/svelte.js')
```

Also if your solution for component is just re-exporting trans functions, could we just avoid it and import them from our alias?
For example: (using with previous alias example)
```html
<script>
    import {trans} from 'lingua-js'
</script>

<h1>{trans('Hello World')}</h1> 
<h1>{t('Hello World')}</h1>
<h1>{__('Hello World')}</h1>
``` 

Maybe react version will also just import then the functions, cause (as far as I know) we do not need to extend anything in React as we do that in VueJs, and people will use "pure" functions in their JSX files.

Maybe then we can create some common name file, that re-exports functions just to make things clear.
And in next phase (before v1 release) make a monorepo here, and publish an npm package for this, to avoid alias binding for users.

**Comment by mikield** [comment]
@sergix44 could you remind me what this pull request was for?
I have lost the main point of our conversation.

Is this PR actual or it's outdated for now?
